wagtail/admin/templates/wagtailadmin/pages/listing/_list.html

```diff
@@ -17,32 +17,6 @@
         {% block pre_parent_page_headers %}
         {% endblock %}
```

Comment:
Investigating this top block, it seems to only be used in the `wagtail/admin/templates/wagtailadmin/pages/listing/_list_move.html` template, which is unused (should have been removed in #8689).

Would you be ok to remove that template, and this block? Looks like the following are also unused and should also be removed:

```
wagtail/admin/templates/wagtailadmin/pages/listing/_navigation_move.html
wagtail/admin/templates/wagtailadmin/pages/listing/_page_title_move.html
```

And finally there is another block that does nothing, `{% block pages_listing_title_extra %}{% endblock pages_listing_title_extra %}`, that can also be removed.

---

This might feel a bit unrelated to the PR but those are the exact templates I was worried about us affecting with this work – so I think it’s worth removing all of this at the same time (or can be a separate PR if you prefer).